addons/resource.language.en_gb/resources/strings.po

```diff
@@ -2860,59 +2868,59 @@ msgstr ""
 
 #: xbmc/settings/AdvancedSettings.cpp
 msgctxt "#669"
-msgid "Verbose logging for SMB library"
+msgid "Verbose logging for \"SMB\" library"
 msgstr ""
 
 #: xbmc/settings/AdvancedSettings.cpp
 msgctxt "#670"
-msgid "Verbose logging for CURL library (http, dav)"
+msgid "Verbose logging for \"libcURL\" library \"HTTP(S), DAV\""
```

Comment:
If it were up to me I'd leave `CURL` as it is too. People don't generally talk about "libcurl", they talk about "CURL".